data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java

```diff
@@ -32,15 +32,111 @@
     })
 public abstract class AbstractFlowThreshold extends AbstractThreshold {
 
+    /**
+     * Side of the network element which is monitored
+     */
     protected Side side;
+
+    /**
+     * Direction in which the flow is monitored
+     */
     protected Direction direction;
+
+    /**
+     * maximum admissible flow value
+     */
     protected double maxValue;
```

Comment:
I think it would be better to add a minValue attribute

Reply:
As discussed, the FlowThreshold can for now be : 
* monitored in one direction (Direction.RIGHT or Direction.LEFT) with one maxValue
* monitored in both directions (Direction.BOTH) with symmetric max values

If the need for asymmetric bidirectional Threshold appears, we will create a new AbstractFlowThreshold implementation